### PR TITLE
Support some comments before 'package'

### PR DESCRIPTION
**Pull request by zchee**
Support some comments before 'package', such as License comment.

I don't know whether the correct way, but can leave the license comment and support the empty line before the 'package' `token.Pos`.
It may be ugly implementation. Please point out if there is a mistake.

Fixes https://github.com/cweill/gotests/issues/37

### CODE REVIEW

**Comment by zchee** [comment]
Also, Should I add the that's situation test case?

**Comment by cweill** [comment]
Hi @zchee, thank you for this pull request. Please add at least one test case to verify this new code. You can add it to `gotests_test.go`

The test case could look something like: 

```go
...
{
  name: "Existing test file with package level comments",
  args: args{
    srcPath: `testdata/test_existing_test_file_with_comments.go`,
  },
  want: mustReadFile(t, "testdata/goldens/existing_test_file_with_package_level_comments.go"),
}, 
...
```

If you just create `testdata/test_existing_test_file_with_comments.go` and run `go test ./...`, the golden file will be output to your `/tmp` directory and will print out the file name to the console. You can just copy the file over to the `goldens` directory, and then your test should pass.

**Comment by zchee** [comment]
@cweill I got it, I'll add the new test case later.

**Comment by zchee** [comment]
@cweill I have added test case. Is it correct?

**Comment by zchee** [comment]
Sorry, renamed test files `name` and testdata file name to your suggested names.

File: testdata/goldens/existing_test_file_with_package_level_comments.go [hunk 1]
```diff
@@ -0,0 +1,21 @@
+// Copyright 2017 The Go Authors. All rights reserved.
```
Comment (cweill) [inline testdata/goldens/existing_test_file_with_package_level_comments.go]: Please update these test comments to some Lorum Ipsum. We probably shouldn't have real comments mixed in our test cases.

**Comment by zchee** [comment]
@cweill Thanks for code review.
Got it. Will do :)

**Comment by zchee** [comment]
@cweill I have fixed license comment to Lorum Ipsum example comment.
PTAL.

**Comment by cweill** [comment]
LGTM. Thanks for this change, @zchee !

**Comment by zchee** [comment]
@cweill You too! Thanks for the great tool. gotests is game changer :D

**Pull request merged**